apps/arena/lib/arena/game/player.ex

```diff
@@ -189,19 +189,39 @@ defmodule Arena.Game.Player do
           skill.activation_delay_ms
         )
 
-        action_name = skill_key_execution_action(skill_key)
+        action =
+          %{
+            action: skill_key_execution_action(skill_key),
```

Comment:
yeah that makes sense , so we can avoid this abomination `action.action`

Reply:
actually i think this is a bigger change than just renaming this, because we need to change the player action everywhere. This change was only something to avoid adding multiple parameters to the add_action function, if you think this make things worse we can revert it

Reply:
Oh, let's leave it like this then